packages/teams-js/src/internal/telemetry.ts

```diff
@@ -113,6 +113,8 @@ export const enum ApiName {
   ExternalAppAuthentication_AuthenticateWithPowerPlatformConnectorPlugins = 'externalAppAuthentication.authenticateWithPowerPlatformConnectorPlugins',
   ExternalAppCardActions_ProcessActionOpenUrl = 'externalAppCardActions.processActionOpenUrl',
   ExternalAppCardActions_ProcessActionSubmit = 'externalAppCardActions.processActionSubmit',
+  ExternalAppCardActionsForCEC_ProcessActionOpenUrl = 'externalAppCardActions.cec.processActionOpenUrl',
```

Comment:
take `'externalAppCardActions.cec` implies that cec is a sub capability in `externalAppCardActions` but that is not true. What about `externalAppCardActionsForCec.processActionOpenUrl`

Reply:
Sure! I will change to `externalAppCardActionsForCEC` to keep align as your AuthCEC changes.